README.md

```diff
@@ -198,8 +198,8 @@ import MacroTesting
 
 @Suite(
   .macros(
-    record: .missing // Record only missing snapshots
-    macros: ["stringify": StringifyMacro.self],
+    record: .missing, // Record only missing snapshots
+    macros: ["stringify": StringifyMacro.self]
```

Comment:
Thanks! We actually just reordered these to make things read a little nicer (no repeated `macros(macros:)`).

```suggestion
    ["stringify": StringifyMacro.self],
    record: .missing // Record only missing snapshots
```